man/man1/p11sak.1.in

```diff
@@ -238,11 +238,52 @@ attribute of the key and
 can be used to set the binary attributes of the key (see below for detailed description of the attributes).
 .
 .PP
+.SS "Generating IBM Dilithium keys"
+.
+.B p11sak
+.BR generate-key | gen-key | gen
+.BR ibm-dilithium
+.B \-\-slot
+.IR SLOTID
+.B \-\-pin
+.IR PIN
+.B \-\-label
+.IR LABEL
+.B \-\-attr
+.IR [M R L S E D G V W U A X N T]
+.B \-\-custom\-\attr | \-c
+.IR [[pub_attrs]:[prv_attrs]]
+.B \-\-help | \-h
+.PP
+Use the
+.B generate-key
+.B ibm-dilithium
+command and key argument to generate an IBM Dilithium key, respectively. The
+.B \-\-slot
+.IR SLOTID
+and
+.B \-\-pin
+.IR PIN
+options are required to set the token to
+.IR SLOTID
+and the token PIN. The
+.B \-\-label
+option allows the user to set the
+.IR LABEL
+attribute of the key and
+.B \-\-attr
+.IR [M R L S E D G V W U A X N T]
+can be used to set the binary attributes of the key (see below for detailed description of the attributes). 
+.B \-\-custom\-attr | \-c 
+.IR [[pub_attr]:[prv_attr]]
+can be used to set the CK_BBOOL and CK_ULONG attributes of the key (see below for detailed describtion ans syntax).
+.
```

Comment:
description

Reply:
removed, this is part of another commit, my bad